### PR TITLE
test(vrt): add for button

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -13,3 +13,25 @@ In a new terminal window start the tests using:
 ```sh
 yarn e2e:test
 ```
+
+## Running end to end tests against Sauce Labs
+
+```sh
+yarn e2e:build && yarn e2e:serve
```
Comment (blackswanny) [inline TESTING.md]: can we combine these into one command?

File: e2e/helpers/index.js [hunk 2]
```diff
@@ -11,7 +11,25 @@ function goToUrl({suite, test, client}) {
   return client.url(getUrl({launchUrl: client.launchUrl, suite, test}));
 }
 
+function formatFileName(testName) {
+  return testName.toLowerCase().replace(/ /g, '-');
+}
+
+function assertVisuals({client, id}) {
+  const isChrome = client.options.desiredCapabilities.browserName === 'chrome';
+  const isMac = client.options.desiredCapabilities.platform === 'macOS 10.12';
```
Comment (blackswanny) [inline e2e/helpers/index.js]: this is very exact version, may not be the same for other machines. Btw, what the Sauce Labs runs on?

Comment (gergelyke) [reply e2e/helpers/index.js]: what do you mean by `may not be the same for other machines`? here we request a chrome under macOS 10.12 - or are you referring to virtualization? if so, I don't know, but I guess that does not matter, does it?

File: TESTING.md (hunk 1 above)
Comment (KevinGrandon) [reply TESTING.md]: My recommendation: document how to do this with the docker command, so we can run tests as they are run in CI.

**Pull request merged**